freebasics/templates/core/banner_page.html

```diff
@@ -0,0 +1,7 @@
+{% extends "base.html" %}
+
+{% load core_tags %}
+
+{% block content %}
+  {% bannerpages %}
```

Comment:
I think you're meant to be rendering `self`. Since this will be used in the preview, wagtail does some special stuff.
For example, you can preview something that's in draft, so our tag will not be able to handle that.